inc/databaseinstanceinjection.class.php

```diff
@@ -64,11 +64,13 @@ public function getOptions($primary_type = '')
     {
 
         $tab                 = Search::getOptions(get_parent_class($this));
+        $tab[5]['linkfield']    = 'items_id';
```

Comment:
I have a bad feeling about this.
If the output of $tab changes in the future and $tab[5] is no longer what you expect, it could be very weird and difficult to debug.
Try to make something more robust, or at least comment for us in the future ;)